app/population/view/daily.njk

```diff
@@ -0,0 +1,116 @@
+{% extends "layouts/base.njk" %}
+{% from "panel/macro.njk" import appPanel %}
+{% from "data/macro.njk" import appData %}
+
+{%  macro statsEntry(property, value) %}
+  {%  set key = "population::daily_view.labels." + property %}
+  {%  set label = t(key) %}
+
+  <h4 class="govuk-heading-s">{{ label }}</h4>
+  <p class="govuk-body">{{ value }}</p>
+{% endmacro %}
+
+{% macro statsGroup(headingProperty, stats) %}
+  {%  set key = "population::daily_view.labels." + headingProperty %}
+  {%  set label = t(key) %}
+
+  <h3 class="govuk-heading-m">{{ label }}</h3>
+
+  {%  for stat in stats %}
+    {{  statsEntry(stat.property, stat.value) }}
+  {% endfor %}
+{% endmacro %}
+
+{% macro transfersGroup(headingProperty, transfers) %}
+  {%  set key = "population::daily_view.labels." + headingProperty %}
+  {%  set label = t(key) %}
+
+  <h3 class="govuk-heading-m">{{ label }}</h3>
+
+  <p class="govuk-body">
+  {%  for transfer in transfers %}
+    {{ transfer.count }} from {{  transfer.establishment }}<br>
+  {% endfor %}
+  </p>
+{% endmacro %}
+
+{% set displayDate = details.date if details.date else date | formatDateWithRelativeDay %}
```

Comment:
This ternary looks like a hangover from the previous implementation. `details` is gone so this could become:

```nunjucks
{% set displayDate = date | formatDateWithRelativeDay %}
```

Reply:
Removed the usage of `spaces.details.date` and simplified this line as suggested.